VoxelSniperCore/src/main/java/com/github/kevindagame/brush/multiBlock/GenerateTreeBrush.java

```diff
@@ -121,10 +121,29 @@ private void leafNodeCreate(final BaseLocation origin) {
     private void createLeaf(final BaseLocation location, int x, int y, int z) {
         if (location.getBlock().getRelative(x, y, z).getMaterial().isAir()) {
```

Comment:
Not sure if this works correctly in every scenario, since we might have already created an operation to create a log block at the same position. Not entirely sure if that is possible, we if can do anything about it.